cmd/commands/cli/command.go

```diff
@@ -106,18 +108,56 @@ var versionSummary = metadata.VersionAsSummary(metadata.LicenseCopyright(
 	"type 'license --conditions'",
 ))
 
+func (c *cliApp) handleTOS(ctx *cli.Context) error {
+	if ctx.Bool(config.FlagAgreedTermsConditions.Name) {
+		c.acceptTOS(ctx)
+		return nil
+	}
+
+	agreedC := config.GetBool(config.FlagTermsConsumerAgree)
```

Comment:
do we really have all these different TOS for use? Is it not a single one?

Reply:
Yes that something I would like to discuss and figure out as well. Do we need to make them different? I was sort of looking in to NodeUI/Desktop app, etc. and tried to keep the same consistency.

So lets say you're using only the desktop app and nothing else, well then I guess you need to only agree to the Consumer terms of service, but if you later fire up `myst service` you would have to agree to Provider terms of service aswell.

I would like to hear what @chompomonim  thinks about this, because dropping the 2 different flags and having only 1, would make life a lot easier.

Reply:
@tomasmik, we agreed not to have terms for each small command, but rather just for.
And give to @chompomonim to test and feedback.

```
myst service --accepted-terms-of-service
myst connection up --accepted-terms-of-service
```

Reply:
@Waldz and we dont. in a way. services launch just as you described there.